Add ExpressVPN

spazvt opened the pull request:
* add ExpresVPN service provider, originally added by @deadr0ck to fork
https://github.com/acceptbitcoincash/acceptbitcoincash
* Also add Facebook handle for GoGetSSL.

spazvt committed:
add ExpresVPN from 'acceptbitcoincash'

add ExpresVPN service provider, originally added by @deadr0ck to fork
https://github.com/acceptbitcoincash/acceptbitcoincash

Also add Facebook handle for GoGetSSL.

Carlgo11 commented on _data/security.yml:
```diff
@@ -71,6 +71,16 @@ websites:
+      software: Yes
+      hardware: Yes
+      twitter: expresssvpn
+      facebook: ExpresssVPN
```
Twitter and Facebook tags shouldn't exist when tfa = yes

spazvt committed:
ExpressVPN doesn't appear to support 2FA

On further inspection, the article I linked was a general post about
2FA, not instructions for using 2FA on ExpressVPN's site. I searched
their support forum and FAQ and didn't find any 2FA help or
instructions, so I don't think they support 2FA.

spazvt committed:
capitalize ExpressVPN twitter handle

per recent guidance/corrections.

spazvt committed:
change ExpressVPN logo to non-text favicon

change ExpressVPN logo to non-text favicon per mxxcon's suggestion.

Carlgo11 commented on _data/security.yml:
```diff
@@ -82,6 +89,7 @@ websites:
       img: gogetssl.png
       tfa: No
       twitter: gogetssl
+      facebook: gogetssl
```
Why is GoGetSSL changed in this PR?

spazvt replied:
mainly because when I broke one large PR into 13-15 smaller PRs, I grouped changes that were close to each other in the same file to reduce the number of PRs.

Since this is an issue, I'll remove it and make the `gogetssl` change a different PR (#3195).

spazvt committed:
remove gogetssl change from ExpressVPN PR

mxxcon reviewed (approved):
@spazvt there was no need to dismiss my review... 😕

jamcat22 commented on _data/security.yml:
```diff
@@ -71,6 +71,13 @@ websites:
       hardware: Yes
       doc: http://www.entrust.net/certificate-services/cms-authentication.htm
 
+    - name: ExpresssVPN
```
Name has an extra "s" inserted.
Line should be `- name: ExpressVPN`

jamcat22 commented on _data/security.yml:
```diff
@@ -71,6 +71,13 @@ websites:
+      img: expressvpn.png
+      url: https://www.expressvpn.com
+      tfa: No
+      twitter: ExpresssVPN
```
Twitter has an extra "s" inserted and is not capitalized according to their Twitter username.
Line should be `twitter: expressvpn`

jamcat22 commented on _data/security.yml:
```diff
@@ -71,6 +71,13 @@ websites:
+      url: https://www.expressvpn.com
+      tfa: No
+      twitter: ExpresssVPN
+      facebook: ExpresssVPN
```
Facebook has an extra "s" inserted.
Line should be `facebook: ExpressVPN`
